### PR TITLE
Fix bivariate

### PR DESCRIPTION
**Pull request by yuqizhang99**
## Functionality

fix the function of automatically determining the number of bins for temporal chart

### CODE REVIEW

**Comment by willeppy** [comment]
Can you rebase off main so that ONLY the new commit(s) are in this PR? 

When you make the change make sure you branch off main rather than an old branch

File: digautoprofiler/profile_lib.py [hunk 1]
```diff
@@ -306,19 +296,3 @@ def getTempAggrData(dfName: pd.DataFrame, tempColName: str, quantColName: str, a
         tempAggrData = groups.max()
     print(json.dumps({"data":tempAggrData.to_dict(), "timestep":timestep}))
     
-
-
-def hampel(colData: pd.Series, k=7):
```
Comment (willeppy) [inline digautoprofiler/profile_lib.py:311]: Rebase off main, this should not get deleted

File: digautoprofiler/profile_lib.py [hunk 2]
```diff
@@ -282,7 +272,7 @@ def getTempAggrData(dfName: pd.DataFrame, tempColName: str, quantColName: str, a
     offsetAliases = ['Y','M','W','D','H','T','S']
     i = 0
     previousLen = -float("inf")
-    currentLen = 0
+    currentLen = len(dfName[tempColName].dt.to_period(offsetAliases[i]).unique())
```
Comment (willeppy) [inline digautoprofiler/profile_lib.py:275]: What is this doing?

File: digautoprofiler/profile_lib.py [hunk 3]
```diff
@@ -254,16 +253,7 @@ def getTemporalMeta(colData: pd.Series):
         result = "descending"
     else:
         result = "noSort"
-    vc = colData.value_counts()
```
Comment (willeppy) [inline digautoprofiler/profile_lib.py:257]: Rebase off main, this should not get deleted

**Review by willeppy — CHANGES_REQUESTED** [review]

File: digautoprofiler/profile_lib.py (hunk 2 above)
Comment (yuqizhang99) [reply digautoprofiler/profile_lib.py:275]: I find the initial value of `currentLen` is wrong, which is the reason why the temporal chart sometimes can not determine the number of bins well

**Pull request closed without merging**